### ISSUE TITLE
Hide "Featured" section on Home Page if there are no cards

### ISSUE THREAD

**Issue by devinbalkind** [consul/consul#2890]
### User Story

As an admin, I want the "Featured" section to disappear from the home page if there are no cards in it.

### Why is this Needed?

If there are no cards then there is nothing to "feature" on the home page so the display of that section heading looks like an error.

### Description

*Type*: Bug

### Current Behavior
Currently if there are no cards on the homepage it still says "Featured"

![image](https://user-images.githubusercontent.com/444311/45330470-47587200-b52b-11e8-84a4-855ad60fbfdf.png)

### Expected Behavior
Hide the entire section if there are no cards

![image](https://user-images.githubusercontent.com/444311/45330538-88e91d00-b52b-11e8-8119-e93c2c7326d5.png)

### Steps to reproduce
See home page.

### Definition of Done
Section is hidden when there are no cards.

**Comment by papayalabs** [comment]
Working on this issue

**Comment by papayalabs** [comment]
Hi @mlovic, could you approve this to work on it. Thanks!

**Comment by decabeza** [comment]
Hi @papayalabs, thanks for find this, and of course! you can work on it! 😌 

Remember to hide the hole container:

```erb
<div class="small-12 column <%= 'large-8' if feed_processes_enabled? %>">
  <%= render "cards" %>
</div>
```

😉

**Comment by papayalabs** [comment]
Hi @decabeza, excellent! Ok, thanks for the tip. :)

**Comment by javierm** [comment]
Closed via #2899. Thanks @papayalabs!
